mbed_host_tests/host_tests_plugins/module_reset_mps2.py

```diff
@@ -67,12 +69,18 @@ def execute(self, capability, *args, **kwargs):
         @return Capability call return value
         """
         result = False
+
+        disk =  kwargs.get('disk', None)
+        target_id = kwargs.get('target_id', None)
```

Comment:
Lets add the same error checking that's in the copy plugin for these parameters here.